runelite-client/src/main/java/net/runelite/client/plugins/worldmap/WorldMapPlugin.java

```diff
@@ -224,6 +253,47 @@ public void onXpChanged(ExperienceChanged event)
 		}
 	}
 
+	@Subscribe
+	public void onGameStateChanged(GameStateChanged event)
+	{
+		if (event.getGameState() == GameState.LOGGED_IN)
```

Comment:
I wold flip this if (e.g chnage it to early return if hgamestate is not logged in